### ISSUE TITLE
Import path should be "github.com/sirupsen/logrus", not "github.com/Sirupsen/logrus"

### ISSUE THREAD

**Issue by KateGo520** [honeycombio/honeyaws#76]
## Background
As [README](https://github.com/sirupsen/logrus/blob/v1.0.3/README.md) of logrus said, `github.com/Sirupsen/logrus` is the old path, anything using it should be updated to import and require `github.com/sirupsen/logrus`.

> Some environments experienced problems with the upper-case variant, so the lower-case was decided. Everything using `logrus` will need to use the lower-case: `github.com/sirupsen/logrus`. Any package that isn't, should be changed.

GO111MODULE=on, and $go get -u github.com/honeycombio/honeyaws/cmd/honeyelb :
```
$go get -u github.com/honeycombio/honeyaws/cmd/honeyelb
go: found github.com/honeycombio/honeyaws/cmd/honeyelb in github.com/honeycombio/honeyaws v0.0.0-20200402023807-0968cbcfd4eb
go: golang.org/x/crypto upgrade => v0.0.0-20200728195943-123391ffb6de
go: golang.org/x/sys upgrade => v0.0.0-20200805065543-0cf7623e9dbd
go: github.com/aws/aws-sdk-go upgrade => v1.33.20
go: github.com/honeycombio/honeytail upgrade => v1.1.4
go: github.com/honeycombio/libhoney-go upgrade => v1.12.4
go: github.com/jessevdk/go-flags upgrade => v1.4.0
go: github.com/Sirupsen/logrus upgrade => v1.6.0
go: github.com/jmespath/go-jmespath upgrade => v0.3.0
go get: github.com/Sirupsen/logrus@v1.0.3 updating to
        github.com/Sirupsen/logrus@v1.6.0: parsing go.mod:
        module declares its path as: github.com/sirupsen/logrus
                but was required as: github.com/Sirupsen/logrus
```

## Solution
Replace all the import paths, change `"github.com/Sirupsen/logrus"` to `"github.com/sirupsen/logrus"`.
Where did you import it: https://github.com/honeycombio/honeyaws/search?q=github.com%2FSirupsen%2Flogrus&unscoped_q=github.com%2FSirupsen%2Flogrus

**Comment by KateGo520** [comment]
@paulosman  @ianwilkes  Could you help me review this issue? Thx :p

**Comment by paulosman** [comment]
This has been fixed in #86  thank you!
